File: soolicious/src/main/java/com/group5/soolicious/Inventory/Inventory.java
package com.group5.soolicious.Inventory;

import java.util.Date;

import com.group5.soolicious.employees.Employee;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Inventory {
    private Integer id;
    private Ingredient ingredient;
    private Date datePurchased;
    private Double quantity;
    private Date expiryDate;
    private Double price;
    private Employee employee;
}

File: soolicious-angular/src/app/shared/models/inventory.ts
import { Employee } from "./employee";

export interface Inventory {

    id?: number;
    ingredient?: Ingredient;
    datePurchased?: Date;
    quantity?: number;
    expiryDate?: Date;
    price?: number;
    employee?: Employee;
}

export interface Ingredient {
    id?: number;
    name?: string;
    description?: string;
    quantity?: number;
    measurementUnit?: number;
}